queries.sql

```diff
@@ -10,3 +10,53 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* transactions */
+/* Transaction1 */
+BEGIN;
+Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+/* Transaction2 */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+COMMIT;
```

Comment:
- [ ] Just like my previous comment, it will be good if you verify the change was persisted after the commit.

Example:
```sql
SELECT species from animals; -- verify that change was made
COMMIT;
SELECT species from animals; -- verify that change persists after commit
```